### ISSUE TITLE
Windows 11: Display Settings, Scale 150%. Not resizing vertically.

### ISSUE THREAD

**Issue by aksn1p3r** [PathOfBuildingCommunity/PathOfBuilding#4184]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

When I have my Windows 11 Display settings, set to a scale of 150% on a 1920 resolution, PoB does not want to resize vertically.
There is an overflow of the app beneath my taskbar.

<img width="960" alt="Screenshot 2022-02-18 142215 POB 150% windows cant resize " src="https://user-images.githubusercontent.com/52001/154684218-207b0d39-7a4b-4c44-b384-f01796ac7f24.png">

### What is the actual behaviour/value?

The app cannot be resized when not maximized. 
Can be maximized to see the full view.

### How to reproduce the issue

Windows 11, Display Settings
Resolution: 1920x1080
Scale: 150%

Manually resizing the window vertically, fails.

<img width="592" alt="Screenshot 2022-02-18 142302 1920 big screen 150% windows 11 " src="https://user-images.githubusercontent.com/52001/154684685-1fd1e6fb-23e6-413a-9379-f2bfe3eed03d.png">

### Build code

```shell
eNrdXVtz28ixfl7-CpSq8mRKmjsGLisp3SwrkWQtKXt3z4trAAxExCBAA6BkJZX_fnoGoAiQBAXRzknl7Lq0FNndM33_ejD0vvvL92niPOi8iLP0aA8foD1Hp0EWxun90d6nu_f7cu8vfx68u1Xl5GN0Mo8T88mfB7-8s6-dRD_o5GiP0z2nVPm9Lj8vRNEvIGqm0nKis_Ra_T3LL7LwaO8mS_We46s0jMvFb0GiiuJGTfXR3tlcJ3FR7jmqCHQani4_qUinKk7HWfBVlxd5Np8d7bE95yHWj9dZCDSX17cfR3eNZeO0uSxs-5d3t4l60vm4VKVTwI-jvWPQXt3rMzWFnyBNJXMQRTCTB5wQTj1B-N7hZt7xTOvwmYcfENFJepvr8yjSQRk_6NM8Lk8nKg2W63XybaBlB2gr-fU8KeNZEuv8mQUfdHJ8WJMvcRftXVaq5Ox2vJSLKcYHnosx_CFiO19WPvOhLsqTBCzaWgExBI7gRBLEZKcWv8XlZJ2XUE4OmCcYp9hl23gv79O41DuoZphvs7jI0h-wyw4KX05nKmlt2MWUHFBCictdF3lbtV1jxpIxdkA8-Cmk17nh03mSQAFocna68jSb-nG64hGPSXYgsecS6bFO5Y5zrT5GVb6MVBjPi4ZPu5iuVapOs6JHkF3FkW5RYtlFej7uJ3IExaMfpdnmrc6hvJX9d_sqhnqFsQ4yKKGvWeM1LA09dltsB87z8S4MOyw0Lpe1kyDa6XX9rUkJ2d5Feaa_N8jYFoEtyu40vkyXaki0RV6T0CXd-3vISmjcL5vG1qzzD7fLTQqO5YFLkIso3tIrrfmhJcdTlVyr7_F0PoXec6e-6uWqFIQcEBdVf-i2CFqu7yK2pUEHbw31ZRo8Mwi2Te6nNNeFzh8ajX3bCm2WOtoaDtyaBSN931DeQA68faUrrYPJBYCgkSqXFqAe9BoqxbZUXaIHjLdayxA3rbW1AGwyFiL9OFZtRUxjcCGWCDP_3SakbTf3YOuKm222Pb7PWgtwfNDddiz9jS5XfInYAfTULiagX_e_Jw68rSyvUh3oz1Od3z-NJ7FOwjZfp_a_qTzsUVYbgnsUUxNWTZZe4bXD5pssr_T6-YMqepW_SpmKuqmH150nGnApcIR6BWej7lkh-7uZFJLXsR3n02yeN4o9p1u1qOibSmDZ2W9uJ09FHEBm2DlppMN50OoX3Uj1eey5zh70FDLeTk0wsTWmE0q6BwIY9_paACQnyas4jstSBV_PsvBev2qR13OM57MZlD4TN3353sc5WLqIG4ile5ZYEn-EifZUzfogdJNaLfmUv0zcW_5VfD8pU5gXVhYRbk-O_ppMVFasrLIv-pD3XuI9jD695p7b7BFETsxxSPE6akBFS0d07iPX6T-eestvkfda4DwN57kJ0N5rrHL0WubY1gTonnp6FU_jBmayHO8O7QmTeQXTapaXTqKK8jgIsnlaflDF5GjPF4wQ5Cvp-kIImFyZJlRIGjDBXeZhTbEfUQ8SzbCOtEqmkIsfb_6o3jCbVUGp80oaC2UkNVGaahzCfMqRDvwgkr7GAWHEhc8wJ34Vuu9OVRIUVrHLdDYvndQeUk3jIvjiz6PInEftgaK5PUc7_Xh9cnxXG6JJX3yNk-RLOp_65qym-q8506oox9qWWCfIkkTNCh0e7UUqKUBwDC_HhncMvSUoe9ED1q7PuPpQAw4I8hic34fYnOz02rI9LutDaY6xehFWJxn9DXb3NNMmCIu-NqtrdR9qe_7Ui7I6MOqnoD2Y6ucGHainXpTP4KgX9Xmij-PEdO5-VruGLKiafj966MF57M_LntFmh79e-zBjTy8Fm3C2Z3b0I6whWa9N1Ai0D23d6foFJaCh3m4705GGcO-Z9racVAN8VbIXVfGdzbXCKaBuX-hpcfIEnfO9Me_KseBUlRCI0AHMc4S77LkkX1XPFcp8bspoU8rizVBHap6Y93-dqyQun0x_arxbSyDwZjHJHg36qsSYClCAZ66uqk-Ok7KWYNZYKFipb7SwzxyqZmV_tzqaLuXEaZDMQ32Z1tDg2TiJ8s3ae869eUhxajrW0V4aJ-bZivITY8ZasyQzM5dWMzAy3ltdyqxiNvLLO9hazXuRZL5KyHKtSk9M91oE-HkJI-gyrJs0tKPT_GnP-VZpfGmrhjUZbFZPze_XulShKtXhJXTm4tAY5dDuBl5tkgFwbk2voFIZNDJ9zgwby_Udw3z409S6zpJSp-MJIOyd1doko69aFa9TMVdZU8n9r4kga0pzLpelK9tbOIG85IQPOldJ-DG6BQ-XT-sirbRXOmdV5rdGmvfwS8XuZJGzYVOQ_t_j4hzk6TGMNTFgwHo1O4D8f_UjesmP4_l0mqUj6BgXWaKnP8eRiyxbE_5Kj1b8zrjMUu2s729BoENLURH0rjP4JdOcaWhL0zhV1anHjjbYLKWvBdrc_6kgPcnCJ2dx1vQDHetFk58-BYl96r-jsVf5-5p5wffzgqdGH_YksH2Q9nrlnoHMNml9VbUynIUQpzkg9lDc7an3ST4v7bZ-QNt1GX11XHL2dijpqRhUmnx6ovLc3rXYWbeNYvqqd5lGc0DmjhlUU20fz_9EeFVv0sxcpTkl21nHVQmN6Nmq3oLvpydjfXKwu0YrAnr7q3li8Z-o3hmM5T9Qt8VL9r3Rj7cTVejRfPc-uUlGXwNbRsdw9o0Z1jNkTlVRfkybY-_u0dMtq3ebAgnO4wSGj0qKs5zEeyXKS0pfAprKoXJCDO4-Wm0U0jf3F8yO5f5pip0kWRaOdmq-tVbrEpqFZ3s3MqzOaHkO-39fAj7oZKrLfy92g6FLF6XWyYnpSvnugDnXKlyR0RsuG16nZv7vNbZ8ydafdXqvqyeKO1p5XUJj2a02XnL2Ts8Xq-1nZS5FzoFiXObx1x9Qq0tQ3wgyApxKglOL-HnQahTPsqLcXbtV_r5KLfj-UxkB9njQxb-3_BybQ2xA1fbx4jhR052tXEnKVWKeiE1iHb3a3s8SnGcRP62VvQfV9JkqJjvrty6hr16W07GsP3tgtDfYre9-ZD5eF9J7LK5u0AOkMuzFzx8-RvNykuii-AH11kT0zv8F40oBqF_Z5zd3udaOqpLTMuH6iQj80vxuhtkmMqkMBSV_qh_6He39k3LJvSGngrj_Gv5TEC7lkLmCEfiNcfh0yD2Pin_BvrLQPI6hrkTIkAwJQ5IOOfbcoYsFGTLpDilCFA2xpIgMMRICD7nAZCgYhw-JQJ4YYpcLYMaMcliXeGRIPM7osFqNMoS8IeEepUPiYoaGAmFuXlOEQY7EIIEhLobVXrGQAg058qQYMspgLcKk4EMPe2woOEawJ8xcDDsTHoYPPcyGzGMgBrbP4B1CQRi8DcsKF97gnovBBpJR0B20MXKpy4aUc2H3R7EhZyDYXD4AZRH8ZMTsG2hgQeZSCcIwAaNU9mVEUKBnoMvQEx43L8EImBpOKYEHI-SClp7Rm2PJYXgrwbWNb-lgt_4CjnElqyvweRiXOrwxrqlD_9Poyr74ZVKWs-Lt4eHj4-PBTJWTLNLf40QfBNn0cAZiIGL2bbTvm4UOj-Gfk_tfj2__52N2HH7X599HNxdfshNOP38cUTabjX4vvpyXiTwf60_uw69XH27SYizOyW-fb9T93bf9374lX-6OibgdyV_Dp1lycT653M_3P9H8Idy_mXwqzi9Tlvz--erL54vk99koR-T75d9mfpY_Tj7uf9nnRTmaTr9e_PHtH_MRfbjLHu9mGfnwt_Hv0YRk7698kf4VJ8gNvv16fHz77fLqfD6f5Ci6ee_-_eT6yCp_uND-XfXdo2KROZANNnFMupgXN1Au7K0Q8-biF5tMn2P96BTaNAAAEqZZ_iPLpn8c7e1jKuWBBJ9jcJnk1UPJD1qV12q2SGNDe7U8m7IPNEtVnsWQcLl9YNuk_N1c8vUOIBwhSwSi9eW6d7aM1FltXo91VRrmha4uZ1ePH-zbjaeghtQ-7yVVdIyUeZ7y1hkdj84Hd1luLgY4oyz7OjjN8hxiJoSy-ahzp3qgPqiOit86nNQvT2A-ry__QuS8ddABpcIbfErjb3PtXJ69dYirWaAIJl7k45ByFghXR5J6kRtJElBNGdcRkjwKhS-JRz2qmQyF4EpS4vsDu2drsbeON6if78JKg9qBsPv90cASjPQ32NvgcjpL4iA2H-HBG4KcMoOyZm-pO-YMavBGmrcqDQbenwwcgvnCnLzV771hazz23qrOVakdfEDte84MLFNYcw-wbIoZl_PUUSlMLOaRvTMCmgcoqjbSjD5tZ3htZ3y6ufz1E7hjAkDVCJyaJ63n6j7Rg7-qUDsfzIN2XTaNLEKCfBmAzUKwJ6bUIxT5IQ2wFzA_AJN7KgioCMJABT4PQsQUD5EvhGJCeVHLyO7SyLxp5AvnomFm0TTzwhmg-oUNzFw7RdXe4C3_qRLtEKe6gmOM-z7RenAchoWDrYMIXz27HrzB6254wyvT3ytz8cDJUudvUKAGGDXNv7id6thbSh1Wpxut_tsEBl3zqLE05jeHuYWKk8EoK40mI-jB4WoyENSZDGxQXwWBSOSL15vIGr6ETEDaCwjzmRtgIahwCQt8j3ihFwaediVBkCqhx1wUMk01fKZNV_V8FXHZ9iVGXRmzLWdEy5orMQweWCYPbVFWtjTWq-pTZMwFvmkRnc1z-9DLkJ3O80IXxotP2Rwk_6kRIHbZgaVw6jsoE_UA2dcWV006FVx5lt3hcryp7H3IUtDEpJrJqwEAe2f8mOVh0ycUBV4YutyD5g1lTDNAENT3wE8-0YGHPGj8CoUqkJpAweMi1FHkawxIQERKoJZPCFv6BC99crE_aiUYhFXLKayldgVKneMgAJWDJzB9CWUCUsaWtjP9HUAcrDBgrfK2mmE2_ywH5ps_xLROTnNnePHJP8FWEeTDv9q1096_3ppzuLPS1Wdp0LV0PoCWGRZTlTrH33XTDX4YaBoFoYwirpF2I8-VYRD4EYBTzLlmKoSfHsHgCS2FAuDFtQZq5QVUCtZyg2g0E9pKDVvr2gnCeLvcYU9ss-sbCNJtHQR60loD-ZO94LAQVBuzPg62l_xgHVVUoni73KlUOebLcEZClSqWInjOpUABMHDspUbTTCDfPsRlh4_YZh8pf54oCLNCDcbx1AxUo8xvecdVgHPdQGniSxVRIkQARSviKAiVxNDivUABdg89rZUbAqRHJFQu9rVJKQVaNb1DN-bIb_vP_7b9sVkVviHdAdjMwBY3WVbowYUJ6uPpPGm3UxdxqXwMGS79SGqAQX4kFHJRAPMKVF2iMfelT4TSNOTE9QIvQkKTAJmvznHajjO3GUdeO6mxeLl-vgFYaUvuSq4Ttgm5YK8ZeCY0BoT2WIQCETA-39p3qouJJoIMHGo23SjLHa2CiXOe6umTM4lL0-KfTBWtwrbo8IfYGFpXAGn0fGZ2dAyt_DKH8BwZBRse0S7SAIRNfFGsQuFGYG5ojABkMIP0hnFN8sBTlErEBOVeiLDvuy44IwoJjVYzv1FmWdsjVdUzpmCraW2dUCv4BhPzq0psqtZ3XgekhQLrYoD7mN84s5Z3nljwAssuXdBlUbcbOhrsMoEppQTw8gTCYMqIoqZNI2rG-NCTinAaIulSErAoAltyLLCWKnCZF4RcISi6HqR1yClyGYyo1HWFbCNz2kxY2upqpqA2DS5XALqBzittf6X8PU4AKDmPBnWZyFS1Lh59sbcR18jkotW-7Ce2r0nRCPj6U0KaUs3tdSu1OjOuNzo4VWmaQdiDnSdG1bDe4yn8TKD9tcuvpVl8ZrMm7vSo3FCzrC-di0yF0ETiFPIvMBAAtvgAchY4FG-ZyqTpfg3fcxcroXwJXdNDCPmuDrDvRoBcVEQA6CjtUhjXFIZZQWAWMI18-JQHMLIFIXHbvpcN37dQpvF-E9NAlLTbqNxUwijeOn15B3Kte77B3OaPdXOjcv3AVIbRztWKU0-av2xE8YgF3KAPjjgM74IwqZVGSEjIIjOcSY_5XPsRTMNh4LpMEh6gsG1d9DOqFdtUrdiGakW8Ph1J7FSt8Ar-uz6-uDwd3CYmTqN5AhGtzDckYDZUxdfFEHYMODdLW0gDwhZiksoIQDn2ONgOIKEvPMU97SNObCz7MN4iolTkEim570WmuHECDTxo2ZeRZfSSlq3ddrC-IdbQ9ffdnfoLUgPG12PYqe9HV4poO5x02WTzHHqik3JhgTMd6PihwsUPT475qGkNBOGloSkCLvFwiDwAxR7AEKloRKG0Q_i5VDIsCOcSu77ULMAB5DmCmsCEu5LLZMsI8obaMIJSqNP7crJSKuuRZLUO_5HNnVJB6TSzYm6-bGpq4PcyV4tAjfJsulpoIWapuz79mzl_a-DdQMcDMFLXWHsQcQnRNY1DqB2dYwnbFJef0jBO5mbsh6qSFQWsZMvOc3SOtUpWMh-BjQlTgsMQGCIVRIALCYq40JEUAYlk6MNoIkIeelhGyg0irIVG8JpID2Ov-7SrUURROy5ZywuLquaYYjkwGVUWzuV0Ok9NItsGW08BBsaJuoAWThw59u5Z1ccWRJsEmKPB-aysTgaycKscZUO_OgNa4evyxQpov_k4uj6-GlyZZHM-PPl5HFYeaJpdBSQEU1NEGEFB5HIEs4irTMDzyIdh3ZVRKKHIEoUMfkRU4yD0iO8JVzGsVgtuL7t37F9siqXTLDURWjrb9eCcYOVxyn1ITJiimAgoNAzqKpiVqC-RDnngUgVgTCrXByqAb76QOkBUwkDV1oP306PdhKuy9mJzdLtHq5M8exxcxIk5PR5DpsJs9QxRGO2AKLhxUgZEHUdluDUTAIiBROPYwxH1XMR0xJgLEyX0VMCuFHCM9KD-Q6slFMOo4AMd5wIAD-QgYp34tQlhLIjZBmHeENI4D6t_q7f_PKmhDS3CAJGart-EVvVc893w1myGWffg9iPwR27Mw5O4mM7LyXroeiKACRngDhUuhZEe2osfURpQj1IsXMG5D_2YeZ7kEsAkdj0E5Y9hHHoRDtz2NIHp5tCVvTLQ2xCbz5dbnLtsHkwAUptz3gs1T0sI0AaIFrITRJNW-LmegqISIUbhD9aBj4gMlFYeo4qFOIggNSMME1REXJ94Aj6UXCviK0B7EZaduUpaEPpk5aiWrpxEtXvwM4Qma70Tkx5HtbhPwDwf_wHqbp82thwyTrLFl6OXX4YBeGcfiC_-4poGUdVXaYNGrhMtbpw803jrNAYgNSho11K8QSTWiRbXvZ5p3C5BTb1YFxFrEPEuItIg2mCihR2d8aO5f70g3bAxM5K0pKENdmp--2NByDpkNXQUm0TZq8gLig2Oq07ZliQLE9RPL7c8uWyeHG8yhrEFTAlPFqFVmdN0LVpc-dhkxxU20o9tLYzXbbHjhjbp8sKm7KTwOpbOTNu0J7JpT3Q3-_azAvlJbGy3TYrdVuvJtsma-AUPbwqoFzxQFa_XLrQhlMQu67wQf11FvIdkunse4V00YbsFA94t9NjL_llvbBtM-1q5mwJsR817lZ4NnfflDe1QB-lLej83wFc6cLOZd4owvgtTr6xchy79sEsPTfnOhiU7hKLYLZvobhFMXx9nbPfav0NU452NT_sk5wag2g-pvrjxHwBBPX25ux_EDnG5uyN-IGD61uX2iLA1bzZOE71d84LluiaRHlWG7GzfHStGf4euDUw91KE9DLU-ZvX2A9vZWnyXZtcnYdYmwt5R_sKWuqbJw3qctFd57cmG_RvesjSK79f-yjYzbcbmupi9XmO-bO34WZZoldZXcOu_Canmfne4-r9O-F9TNbDf
```

### Screenshots

<img width="960" alt="Screenshot 2022-02-18 142215 POB 150% windows cant resize 2" src="https://user-images.githubusercontent.com/52001/154685134-8ea90a3c-19c3-4107-87fc-04ebf7e70bcc.png">

**Comment by aksn1p3r** [comment]
Also now noticed cannot scroll horizontally at the Configuration tab, to set the enemy options that are on the far right.
<img width="985" alt="Screenshot 2022-02-19 204858" src="https://user-images.githubusercontent.com/52001/154814770-480c1660-6d9f-42ed-a29d-0cd478558e70.png">

**Comment by tvinki899** [comment]
i have low resolution and i cant even move or see "Enemy stats" category

**Comment by pHiney** [comment]
I can confirm this happens on win10 at both 125% and 150%

I suspect the issue is we have a minimum window size, and the vertical axis hits that minimum instantly

X

**Comment by LocalIdentity** [comment]
Fixed in 2.16.0

**Comment by aksn1p3r** [comment]
Thank you yes, it is now like a holygrail percentage width layout, grids go below if resized to a smaller window
